store/tikv/coprocessor.go

```diff
@@ -56,6 +56,10 @@ type CopClient struct {
 
 // Send builds the request and gets the coprocessor iterator response.
 func (c *CopClient) Send(ctx context.Context, req *kv.Request, vars *kv.Variables) kv.Response {
+	if req.StoreType == kv.TiFlash && req.BatchCop {
+		logutil.BgLogger().Info("send batch requests")
```

Comment:
It looks like a debug code here. We can delete it or change the log level to `debug`.

Reply:
ok, change it to debug